lib/beaker/dsl/structure.rb

```diff
@@ -58,7 +58,7 @@ def step step_name, &block
               else
                 logger.info("Exception raised during step execution and debug-errors option is set, entering pry. Exception was: #{e.inspect}")
                 logger.info("HINT: Use the pry 'backtrace' and 'up' commands to navigate to the test code")
-                binding.pry # rubocop:disable Lint/Debugger
```

Comment:
Please keep this disable statement. We explicitly want pry, so moving it to TODO is counter-productive.

Reply:
We cannot use the statement here because we disable all `Lint` rubocops in `.rubocop.yml`:

```
Lint:
  Enabled: false
```

I would like to remove that from .rubocop.yml in a followup PR and will readd the comment to lib/beaker/dsl/structure.rb.

Reply:
I don't see that, and if we did then I wouldn't expect Lint to show up as a TODO item.

Reply:
fixed now as well